modules/portmaster/src/app/shared/config/generic-setting/generic-setting.html

```diff
@@ -2,12 +2,12 @@
 
   <div class="setting-meta">
     <h3 for="setting">
+      <span class="release-level rejected">
+        <fa-icon icon="exclamation"></fa-icon>
+      </span>
       <span class="release-level beta" *ngIf="setting?.ReleaseLevel === releaseLevel.Beta">Beta</span>
-      <span class="release-level experimental"
-        *ngIf="setting?.ReleaseLevel === releaseLevel.Experimental">Experimental</span>
```

Comment:
Did you mean to remove this?

Reply:
Nope, that must have happened by accident